src/patternfly/components/AboutModalBox/about-modal-box.scss

```diff
@@ -2,113 +2,123 @@
+    --pf-c-about-modal-box--PaddingLeft: var(--pf-c-about-modal-box--sm--PaddingLeft);
   }
 
-  // Element variables
-
   // Brand
-  // Set default padding top, right, left to shared values
   --pf-c-about-modal-box__brand--PaddingTop: var(--pf-global--spacer--2xl);
-  --pf-c-about-modal-box__brand--PaddingRight: var(--pf-c-about-modal-box--PaddingRight);
-  --pf-c-about-modal-box__brand--PaddingBottom: var(--pf-c-about-modal-box--PaddingBottom);
-  --pf-c-about-modal-box__brand--PaddingLeft: var(--pf-c-about-modal-box--PaddingLeft);
+  --pf-c-about-modal-box__brand--PaddingRight: var(--pf-global--spacer--xl);
+  --pf-c-about-modal-box__brand--PaddingLeft: var(--pf-global--spacer--xl);
+  --pf-c-about-modal-box__brand--PaddingBottom: var(--pf-global--spacer--xl);
+  --pf-c-about-modal-box__brand--sm--PaddingRight: var(--pf-global--spacer--3xl);
+  --pf-c-about-modal-box__brand--sm--PaddingLeft: var(--pf-global--spacer--3xl);
+  --pf-c-about-modal-box__brand--sm--PaddingBottom: var(--pf-global--spacer--3xl);
+
+  @media screen and (min-width: $pf-global--breakpoint--sm) {
+    --pf-c-about-modal-box__brand--PaddingRight: var(--pf-c-about-modal-box__brand--sm--PaddingRight);
+    --pf-c-about-modal-box__brand--PaddingLeft: var(--pf-c-about-modal-box__brand--sm--PaddingLeft);
+    --pf-c-about-modal-box__brand--PaddingBottom: var(--pf-c-about-modal-box__brand--sm--PaddingBottom);
+  }
 
   // Close
-  // Set default padding top, right, bottom to shared values
-  --pf-c-about-modal-box__close--PaddingTop: var(--pf-c-about-modal-box__brand--PaddingTop);
-  --pf-c-about-modal-box__close--PaddingRight: var(--pf-c-about-modal-box--PaddingRight);
-  --pf-c-about-modal-box__close--PaddingBottom: var(--pf-c-about-modal-box__brand--PaddingBottom);
   --pf-c-about-modal-box__close--ZIndex: var(--pf-global--ZIndex--2xl);
+  --pf-c-about-modal-box__close--PaddingTop: var(--pf-global--spacer--2xl);
+  --pf-c-about-modal-box__close--PaddingRight: var(--pf-global--spacer--xl);
+  --pf-c-about-modal-box__close--PaddingBottom: var(--pf-global--spacer--xl);
+  --pf-c-about-modal-box__close--sm--PaddingBottom: var(--pf-global--spacer--3xl);
+  --pf-c-about-modal-box__close--sm--PaddingRight: 0;
+  --pf-c-about-modal-box__close--lg--PaddingRight: var(--pf-global--spacer--3xl);
+
+  @media only screen and (min-width: $pf-global--breakpoint--sm) {
+    --pf-c-about-modal-box__close--PaddingRight: var(--pf-c-about-modal-box__close--sm--PaddingRight);
+    --pf-c-about-modal-box__close--PaddingBottom: var(--pf-c-about-modal-box__close--sm--PaddingBottom);
+  }
+
+  @media only screen and (min-width: $pf-global--breakpoint--lg) {
+    --pf-c-about-modal-box__close--PaddingRight: var(--pf-c-about-modal-box__close--lg--PaddingRight);
+  }
+
+  // Close Button
+  --pf-c-about-modal-box__close--c-button--Color: var(--pf-global--Color--100);
+  --pf-c-about-modal-box__close--c-button--hover--Color: var(--pf-global--Color--100);
```

Comment:
Do you need to define the hover color since it doesnt change?

Reply:
nope!